commands/http/handler.go

```diff
@@ -422,3 +426,21 @@ func allowReferer(r *http.Request, cfg *ServerConfig) bool {
 
 	return false
 }
+
+// apiVersionMatches checks whether the api server is running the
+// same version of go-ipfs. for now, only the exact same version of
+// client + server work. In the future, we should use semver for
+// proper API versioning! \o/
```

Comment:
update comment: "checks whether the api client is running the same version" or something